src/google_map_markers.js

```diff
@@ -229,6 +229,7 @@ export default class GoogleMapMarkers extends Component {
     this.dimesionsCache_ = {};
 
     const markers = React.Children.map(this.state.children, (child, childIndex) => {
+      if (!child) return null;
```

Comment:
Also, I can see this library being used with big amounts of markers, so minimising stepts of intermediate array construction (with `.map` and `.filter`), might be a good thing.